xbmc/utils/StringUtils.cpp

```diff
@@ -759,6 +759,61 @@ std::vector<std::string> StringUtils::Split(const std::string& input, const char
   return results;
 }
 
+std::vector<std::string> StringUtils::Split(const std::vector<std::string> &input, const std::vector<std::string> &delimiters, unsigned int iMaxStrings /* = 0 */)
```

Comment:
Isn't SplitMulti a better name? There's already a SplitMulti and it's similar to that since it takes more than one separator